qemu/tests/cfg/sr_iov.cfg

```diff
@@ -29,35 +29,39 @@
             nics = 'nic1 nic2'
             pci_assignable = vf
             image_snapshot = yes
-        - vf_hotplug:
+        - sriov_hotplug:
             type = sr_iov_hotplug
             nics = nic1
             pci_assignable_nic1 = no
             reference_cmd = lspci
-            find_pci_cmd = 'lspci | tail -n1'
             pci_test_cmd = 'echo %s; nslookup www.redhat.com'
             pci_num = 2
-            repeat_times = 2
             wait_secs_for_hook_up = 3
             assign_static_ip = "yes"
+            variants:
+                - vf_hotplug:
+                    repeat_times = 2
+                - vf_hotplug_500:
+                    repeat_times = 500
+                    test_timeout = 10000
+            variants:
+                - with_hugepage:
+                    only hugepages
+                - without_hugepage:
+                    only smallpages
+            variants:
+                - normal:
+                - with_reboot:
+                    vm_reboot = "yes"
+                - with_suspend_resume:
```

Comment:
I think this case should be `- with_stop_resume`,  `vm_suspend = "yes"` oly cause vm.pause(). it is only a stop from monitor.
Another case  ***.vf_guest_suspend.vf_hotpluged  do subtype "guest_suspend" is suspend in fact.

Reply:
Agreed, sure I will make the change.